java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/mavenrepository/MavenRepositoryDeployer.java

```diff
@@ -49,6 +49,8 @@ public MavenRepositoryDeployer(BootstrapMavenContext mvnCtx, String username, St
         this.system = mvnCtx.getRepositorySystem();
         this.session = MavenRepositorySystemUtils.newSession();
 
+        Log.infof("Maven credentials are username '%s', password '%s', and repository '%s'", username, password, repository);
```

Comment:
We should not log the password here.

Reply:
Yeah agreed, I'll remove it. I needed this logging initially as the `kubectl get secret` wasn't showing the trailing newline. Need to find a better way of determining those sorts of problems (even manually via cmd line)